OpenRA.Mods.Common/Warheads/SpreadDamageWarhead.cs

```diff
@@ -56,26 +56,35 @@ protected override void DoImpact(WPos pos, Actor firedBy, WarheadArgs args)
 			var debugVis = firedBy.World.WorldActor.TraitOrDefault<DebugVisualizations>();
 			if (debugVis != null && debugVis.CombatGeometry)
 				firedBy.World.WorldActor.Trait<WarheadDebugOverlay>().AddImpact(pos, effectiveRange, DebugOverlayColor);
-
```

Comment:
Revert this change, removing this white space violates our style conventions

Reply:
Note that `make check` did not complain.